api/grpcserver/mesh_service.go

```diff
@@ -157,10 +156,15 @@ func (s MeshService) getFilteredActivations(startLayer types.LayerID, addr types
 }
 
 // AccountMeshDataQuery returns account data
-func (s MeshService) AccountMeshDataQuery(ctx context.Context, in *pb.AccountMeshDataQueryRequest) (*pb.AccountMeshDataQueryResponse, error) {
+func (s MeshService) AccountMeshDataQuery(_ context.Context, in *pb.AccountMeshDataQueryRequest) (*pb.AccountMeshDataQueryResponse, error) {
 	log.Info("GRPC MeshService.AccountMeshDataQuery")
 
-	startLayer := types.LayerID(in.MinLayer)
+	var startLayer types.LayerID
+	if in.MinLayer == nil {
+		startLayer = 0
```

Comment:
it's 0 by default so you could just do 
go```
if in.MinLayer != nil {
   	startLayer = types.LayerID(in.MinLayer.Number) 
}
``` 
instead of if else